[2023-05-19T17:20:27.152+0200] {processor.py:156} INFO - Started process (PID=19655) to work on /home/julien/airflow/dags/test.py
[2023-05-19T17:20:27.153+0200] {processor.py:758} INFO - Processing file /home/julien/airflow/dags/test.py for tasks to queue
[2023-05-19T17:20:27.153+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:27.153+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:27.172+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:27.169+0200] {dagbag.py:330} ERROR - Failed to import: /home/julien/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/julien/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/julien/airflow/dags/test.py", line 45, in <module>
    dbt_tasks[upstream_node] >> dbt_tasks[node_id]
KeyError: 'source.YoutubeBigData.landing_sources.landing_channels_ranking'
[2023-05-19T17:20:27.173+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:27.203+0200] {processor.py:178} INFO - Processing /home/julien/airflow/dags/test.py took 0.054 seconds
[2023-05-19T17:20:29.193+0200] {processor.py:156} INFO - Started process (PID=19658) to work on /home/julien/airflow/dags/test.py
[2023-05-19T17:20:29.194+0200] {processor.py:758} INFO - Processing file /home/julien/airflow/dags/test.py for tasks to queue
[2023-05-19T17:20:29.195+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:29.195+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:29.207+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:29.207+0200] {dagbag.py:330} ERROR - Failed to import: /home/julien/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/julien/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/julien/airflow/dags/test.py", line 45, in <module>
    dbt_tasks[upstream_node] >> dbt_tasks[node_id]
KeyError: 'source.YoutubeBigData.landing_sources.landing_channels_ranking'
[2023-05-19T17:20:29.208+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:29.226+0200] {processor.py:178} INFO - Processing /home/julien/airflow/dags/test.py took 0.035 seconds
[2023-05-19T17:20:31.520+0200] {processor.py:156} INFO - Started process (PID=19660) to work on /home/julien/airflow/dags/test.py
[2023-05-19T17:20:31.521+0200] {processor.py:758} INFO - Processing file /home/julien/airflow/dags/test.py for tasks to queue
[2023-05-19T17:20:31.521+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:31.521+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:31.530+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:31.530+0200] {dagbag.py:330} ERROR - Failed to import: /home/julien/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/julien/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/julien/airflow/dags/test.py", line 45, in <module>
    dbt_tasks[upstream_node] >> dbt_tasks[node_id]
KeyError: 'source.YoutubeBigData.landing_sources.landing_channels_ranking'
[2023-05-19T17:20:31.531+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:31.547+0200] {processor.py:178} INFO - Processing /home/julien/airflow/dags/test.py took 0.030 seconds
[2023-05-19T17:20:33.191+0200] {processor.py:156} INFO - Started process (PID=19662) to work on /home/julien/airflow/dags/test.py
[2023-05-19T17:20:33.192+0200] {processor.py:758} INFO - Processing file /home/julien/airflow/dags/test.py for tasks to queue
[2023-05-19T17:20:33.192+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:33.192+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:33.205+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:33.204+0200] {dagbag.py:330} ERROR - Failed to import: /home/julien/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/julien/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/julien/airflow/dags/test.py", line 45, in <module>
    dbt_tasks[upstream_node] >> dbt_tasks[node_id]
KeyError: 'source.YoutubeBigData.landing_sources.landing_channels_ranking'
[2023-05-19T17:20:33.205+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:33.225+0200] {processor.py:178} INFO - Processing /home/julien/airflow/dags/test.py took 0.036 seconds
[2023-05-19T17:20:34.791+0200] {processor.py:156} INFO - Started process (PID=19664) to work on /home/julien/airflow/dags/test.py
[2023-05-19T17:20:34.792+0200] {processor.py:758} INFO - Processing file /home/julien/airflow/dags/test.py for tasks to queue
[2023-05-19T17:20:34.793+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:34.793+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:34.802+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:34.801+0200] {dagbag.py:330} ERROR - Failed to import: /home/julien/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/julien/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/julien/airflow/dags/test.py", line 45, in <module>
    dbt_tasks[upstream_node] >> dbt_tasks[node_id]
KeyError: 'source.YoutubeBigData.landing_sources.landing_channels_ranking'
[2023-05-19T17:20:34.803+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:34.818+0200] {processor.py:178} INFO - Processing /home/julien/airflow/dags/test.py took 0.028 seconds
[2023-05-19T17:20:37.092+0200] {processor.py:156} INFO - Started process (PID=19666) to work on /home/julien/airflow/dags/test.py
[2023-05-19T17:20:37.093+0200] {processor.py:758} INFO - Processing file /home/julien/airflow/dags/test.py for tasks to queue
[2023-05-19T17:20:37.094+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:37.094+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:37.112+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:37.111+0200] {dagbag.py:330} ERROR - Failed to import: /home/julien/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/julien/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/julien/airflow/dags/test.py", line 45, in <module>
    dbt_tasks[upstream_node] >> dbt_tasks[node_id]
KeyError: 'source.YoutubeBigData.landing_sources.landing_channels_ranking'
[2023-05-19T17:20:37.113+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:37.134+0200] {processor.py:178} INFO - Processing /home/julien/airflow/dags/test.py took 0.045 seconds
[2023-05-19T17:20:39.988+0200] {processor.py:156} INFO - Started process (PID=19668) to work on /home/julien/airflow/dags/test.py
[2023-05-19T17:20:39.990+0200] {processor.py:758} INFO - Processing file /home/julien/airflow/dags/test.py for tasks to queue
[2023-05-19T17:20:39.991+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:39.991+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:40.072+0200] {logging_mixin.py:117} INFO - [2023-05-19T17:20:40.055+0200] {dagbag.py:330} ERROR - Failed to import: /home/julien/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/julien/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/julien/airflow/dags/test.py", line 45, in <module>
    dbt_tasks[upstream_node] >> dbt_tasks[node_id]
KeyError: 'source.YoutubeBigData.landing_sources.landing_channels_ranking'
[2023-05-19T17:20:40.073+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/julien/airflow/dags/test.py
[2023-05-19T17:20:40.232+0200] {processor.py:178} INFO - Processing /home/julien/airflow/dags/test.py took 0.247 seconds
[2023-05-19T21:20:38.662+0200] {processor.py:156} INFO - Started process (PID=19677) to work on /home/julien/airflow/dags/test.py
[2023-05-19T21:20:38.663+0200] {processor.py:758} INFO - Processing file /home/julien/airflow/dags/test.py for tasks to queue
[2023-05-19T21:20:38.665+0200] {logging_mixin.py:117} INFO - [2023-05-19T21:20:38.664+0200] {dagbag.py:525} INFO - Filling up the DagBag from /home/julien/airflow/dags/test.py
[2023-05-19T21:20:38.721+0200] {logging_mixin.py:117} INFO - [2023-05-19T21:20:38.719+0200] {dagbag.py:330} ERROR - Failed to import: /home/julien/airflow/dags/test.py
Traceback (most recent call last):
  File "/home/julien/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/julien/airflow/dags/test.py", line 45, in <module>
    dbt_tasks[upstream_node] >> dbt_tasks[node_id]
KeyError: 'source.YoutubeBigData.landing_sources.landing_channels_ranking'
[2023-05-19T21:20:38.723+0200] {processor.py:770} WARNING - No viable dags retrieved from /home/julien/airflow/dags/test.py
[2023-05-19T21:20:38.765+0200] {processor.py:178} INFO - Processing /home/julien/airflow/dags/test.py took 0.109 seconds
